saleor/checkout/test_checkout.py

```diff
@@ -36,13 +36,13 @@ def test_checkout_clear_storage():
     checkout.storage['new'] = 1
     checkout.clear_storage()
     assert checkout.storage is None
-    assert checkout.modified == True
+    assert checkout.modified is True
```

Comment:
Are `True` and `False` guaranteed to be singletons in Python? As far as I know only `None` is. This one might be more readable if it was just `assert checkout.modified`.